src/main/kotlin/com/hibob/academy/service/SessionService.kt

```diff
@@ -35,7 +35,6 @@ class SessionService(private val employeesDao: EmployeesDao) {
         val employee =
             employeesDao.getEmployeeByActiveUser(activeUser) //throws if no employee with this id in this company
         if (employee.role != activeUser.role || employee.department != activeUser.department) {
-            throw BadRequestException("The employee info is not accurate (Role/Department)")
```

Comment:
why does this keep showing up in different pr's?

Reply:
I think its a mistake that I done in rebase yesterday

Reply:
I fixed it rn in other small pr and will do rebase for every other branch